docs/customising/make_a_new_theme.md

```diff
@@ -0,0 +1,258 @@
+easiest way is to do it within GitHub.
+
+You'll need your own account on GitHub to do this.
+
+* Go to [github.com/mysociety/alavetelitheme](https://github.com/mysociety/alavetelitheme)
+* Click on the **Fork** button (top right) — if you've got access to multiple
+  accounts/organisations you can choose which one to use
+
+...and it's done!
+
+## 2. Give it a unique name
+
+You must give the repo you've just forked its own name. Ideally, it should
+match the sitename you're planning on running (for example, if your site is
+going to be `abcexample.org`, then a good name would be `abcexample-theme`).
+Don't worry if you haven't really sorted out the domain or even the final name
+yet; this is just so your theme cannot be mistaken for any others.
+
+To do this, go to [GitHub](https://github.com) and find the repo you've just
+forked, and click on its **Settings** tab. You can edit the name of the repo
+there: change it from `alavetelitheme` to your own theme name.
+
+Note: if you're using the Firefox browser, there's a bug with GitHub+Firefox
+here, so you might need to use another browser for this operation.
+
+<div class="attention-box warning">
+  You <strong>must</strong> rename the repo, because you'll run into problems
+  later on if <em>your</em> theme has the same name as the default one,
+  <code>alavetelitheme</code>.
+</div>
+
+## 3. Point your config at your new repo
+
+Now your own theme repo exists, point your Alaveteli install at it by editing
+the
+<code><a href="{{ page.baseurl }}/docs/customising/config/#theme_urls">THEME_URLS</a></code>
+setting in the config file.
+
+Set it to the URL of your uniquely named repo, for example:
+
+    THEME_URLS:
+         - 'git://github.com/yourgithub/abcexample-theme.git'
+
+## 4. Tell Alaveteli to get its theme
+
+Next, tell Alaveteli to pull the repo down from the URL in `THEMES_URL`, and
+install it as the theme your installation will use.
+
+Do this by issuing this command within your Vagrant virtual machine:
+
+    rake themes:install
+
+Alaveteli will connect to GitHub, pull down your theme repo and put it in the
+right place within your Alaveteli installation.
+
+When it's finished, you can see that your theme (that is, all the files from
+the repo) has been reproduced in `lib/themes/`.
+
+## 5. Change the primary colour
+
+You're ready to make a change and see that change appear in your browser.
+
+Alaveteli uses a set of basic <a href="{{ page.baseurl }}/docs/glossary/#sass"
+class="glossary__link">Sass</a> modules to define the layout for the site on
+different device sizes, and some basic styling.
+
+There's more than one way to do this, but the simplest way (which means you
+can most easily see the consequences of any changes you make) is to edit your
+theme's files inside Alaveteli installation, within `lib/themes/`. Within that
+directory, find your theme and edit
+`/assets/stylesheets/responsive/custom.scss`. Find the Sass variable `$primary`
```

Comment:
I think this has been renamed to `$color_primary` (it has changed the banner colour!)